src/CMakeLists.txt

```diff
@@ -2,13 +2,25 @@ add_library(QpSolverCollection
   QpSolverCollection.cpp
   )
 
-target_include_directories(QpSolverCollection PUBLIC
-  ${PROJECT_SOURCE_DIR}/include
-  ${catkin_INCLUDE_DIRS}
-  )
-target_link_libraries(QpSolverCollection PUBLIC
-  ${catkin_LIBRARIES}
+if(QP_SOLVER_COLLECTION_STANDALONE)
+  target_compile_features(QpSolverCollection PUBLIC cxx_std_17)
```

Comment:
Is there a specific reason that `add_compile_options(-std=c++17)` is used for Catkin and `target_compile_features` is used for standalone?